preprocessing/nextclade/src/loculus_preprocessing/config.py

```diff
@@ -109,4 +116,7 @@ def get_config(config_file: str | None = None) -> Config:
         if value is not None:
             setattr(config, key, value)
 
+    if not args.backend_host:  # Set here so we can use organism
```

Comment:
Isn't it enough to just do this check here on config.backend_host and remove the check above?